examples/generative/corrdiff/train.py

```diff
@@ -57,10 +58,9 @@ def main(cfg: DictConfig) -> None:
     OmegaConf.save(cfg, os.path.join(cfg.outdir, "config.yaml"))
 
     # Parse options
-    regression_checkpoint_path = getattr(cfg, "regression_checkpoint_path", None)
+    regression_checkpoint_path = getattr(cfg, to_absolute_path("regression_checkpoint_path"), None)
```

Comment:
Shouldn't this be
```python
regression_checkpoint_path = getattr(cfg, "regression_checkpoint_path", None)
if regression_checkpoint_path:
    regression_checkpoint_path = to_absolute_path(regression_checkpoint_path)
```

Reply:
Similarly for all the other paths in the lines below

Reply:
Fixed